### DISCUSSION TITLE
如何解决段落之间空隙太大的问题？

### DISCUSSION THREAD

**Discussion by stella-zhou** [sjtug/SJTUThesis#931] [提问]
用模板的时候遇到一个问题，就是模板为了让某个小节的标题放到下一页，导致上一页排版过于疏松的。如图所示：
<img width="473" alt="截屏2023-04-20 12 21 47" src="https://user-images.githubusercontent.com/65953185/233260741-c019ba08-7eb7-4e9c-ae70-97830482c223.png">
<img width="488" alt="截屏2023-04-20 12 21 57" src="https://user-images.githubusercontent.com/65953185/233260750-356b3789-4171-4697-b10b-e50c6832d8b0.png">
类似图中这样，第一页每段之间间隙过大，就为了让1.1节的标题放到下一页开头。我想让段落间隙正常一点，有什么措施吗

**Comment by AlexaraWu** [comment]
段落之间的间距由 `\parskip` 控制。默认的 `\parskip` 是弹性长度，值为 `0pt plus 1pt`，有 1 pt 的可伸长量（实际上可以延伸超过 1pt）。

可以设置 `\parskip=0pt` 禁止延伸。

**Reply by stella-zhou** [reply]
谢谢大佬，可以解决间隙小的问题，但是出现了上一页三段很紧凑，三段后面空出几行，1.1这个标题还在下一页的问题。

**Reply by LiShuaiyi** [reply]
我也有这个问题，请问你最后解决了嘛

**Reply by note286** [reply]
@stella-zhou @LiShuaiyi 可以局部设置，也可以全局设置。

# 局部设置
```latex
\section{title}
\allowbreak
```

# 全局设置
```latex
\ctexset{section/aftertitle+={\allowbreak}}
```

**Reply by stella-zhou** [reply]
没有，后来扩充了上一页的文字来减少空隙

**Reply by LiShuaiyi** [reply]
解决啦，非常感谢！

**Comment by note286** [comment]
把`\raggedbottom`放在导言区尾部即可。
